pkg/terraform/gcp/terraform_gcp.go

```diff
@@ -21,15 +21,25 @@ import (
 )
 
 // SetCloudCredentials sets GCP account credentials defined in service account JSON file
-func SetCloudCredentials(rootBody *hclwrite.Body, zone, credentialsPath, projectName string) error {
-	// zone's format is us-east1-b, region's format is us-east1
-	region := strings.Join(strings.Split(zone, "-")[:2], "-")
+func SetCloudCredentials(rootBody *hclwrite.Body, zones []string, credentialsPath, projectName string) error {
+	// set default provider
+	region := strings.Join(strings.Split(zones[0], "-")[:2], "-")
```

Comment:
dont we want to do once per zone?

Reply:
some resources don't have zone, so instead of discovering what zones are avaliable to change them, I just provide default provider - makes things simpler imho